src/test/groovy/io/github/cosmicsilence/scalafix/ScalafixTestConstants.groovy

```diff
@@ -0,0 +1,49 @@
+package io.github.cosmicsilence.scalafix
+
+/**
+ * Created by tomas.mccandless on 2/12/20.
+ */
+class ScalafixTestConstants {
```

Comment:
as suggested in other comments, I think this class could go away if we inlined the build script and created source files as we need them in the individual tests